src/screens/RegisterConsent/styles.ts

```diff
@@ -9,4 +9,7 @@ export default StyleSheet.create({
   },
   scrollViewContentContainer: { flexGrow: 1 },
   content: { margin: spacing.lg, flex: 1 },
+  headerBackButton: {
+    marginLeft: -22,
```

Comment:
Why do we need negative margin?

Reply:
Hello, We are change navigation stack for user redirect to RegisterConsent screen. And next screens (RegisterUsername etc..) are using react-navigation default back button. This back button's default value is negative margin. I don't want to display users different UI, therefore, I gave negative margin. 
P.S. RegisterConsent is our first screen in the RegisterStack, therefore, I can not use react-native default back button.
If you have a different idea, I would like to hear and implement that.